src/TodoListItem.js

```diff
@@ -10,38 +10,27 @@ class TodoListItem extends React.Component {
   }
 
   componentWillMount() {
-    let { complete } = this.props.item
+    const { complete } = this.props.item
     this.setState({ complete })
   }
 
   handleChange = () => {
```

Comment:
Could you guys possibly change the name of this function to something a little bit more descriptive?